feat(116): Add NightVision Icon

louis-bompart opened the pull request:
Hello Cmdr,
First, thank you very much for this project 👌 
Second, here's my modest contribution to address #116 

## What the changes?
Added NigthVision Icon.

## How did you test it?
Built and tested in-game.


## Notes
I made the two images from scratch. I inspired myself from [material.io icons](https://material.io/resources/icons/?icon=bedtime&style=baseline). (no copy, not even with layers.)

## Questions
Should I modify the README.md to include the licensing?

dantman commented:
The `NightVisionToggle` name is based on the identifier Elite Dangerous uses for the associated control in the XML file. So it's preferable to not rename it.

dantman commented:
> Should I modify the README.md to include the licensing?

Yes, please update `Assets/Textures/Icons/README.md` with an entry for these icons. Since they're original I'd prefer if you followed the format/license I use for "Original to this project" icons.

dantman commented:
You're also welcome to update the CHANGELOG.

louis-bompart commented:
>The NightVisionToggle name is based on the identifier Elite Dangerous uses for the associated control in the XML file. So it's preferable to not rename it.

Oh, my bad, I thought it was an inconsistency, that's why I changed that. But if that's come from Elite Dangerous directly, I agree that it should be `NightVisionToggle`. Reverted ✔️ 

`Assets/Textures/Icons/README.md` updated.
CHANGELOG.md updated.

Thanks for your quick-review 🚀

louis-bompart commented:
👋 Hi @dantman, is there anything missing in this PR that prevents it from being merged?

dantman commented:
Sorry I think it's fine, I just haven't spent any spare time on EVRC recently to actually try it.

Though I am curious why Unity still kept the asset name as ToggleNightVision even though you undid the rename.

louis-bompart commented:
> Sorry I think it's fine, I just haven't spent any spare time on EVRC recently to actually try it.
> 
> Though I am curious why Unity still kept the asset name as ToggleNightVision even though you undid the rename.
Good point, I opened the asset in the debug panel and save it again.

What I'm guessing is that I did a rename on it but did not reimport it nor resave it, leading to this discrepancy.
